lib/processors/lib/jaccard.py

```diff
@@ -0,0 +1,133 @@
+import random
+
+def jaccard_rec(current_answers):
```

Comment:
PS: this is the place you use dictionaries instead of having a billion variables.